src/osmTagToNameMapping.ts

```diff
@@ -138,7 +138,7 @@ export const osmTagToNameMapping: Node = {
     clinic: 'Poliklinika',
     doctors: 'Lekárska ordinácia',
     dentist: 'Zubná ordinácia',
-    place_of_worship: 'Miesto uctievania',
+    place_of_worship: 'Chrám/kostol',
```

Comment:
moze byt aj mesita a pod...

Reply:
> moze byt aj mesita a pod...

ten chram by mal byt dostatocne univerzalny:
![obrázok](https://user-images.githubusercontent.com/1406222/121138676-38b45c00-c838-11eb-99b7-b6eaa6529a6d.png)

Reply:
Chrám zahŕŇa aj kostol. Avšak `place_of_worship` môže byť aj na kaplnke, kríži, alebo hociakom mieste kde sa modlí. Takže ideílne len prípadne preformulovať "miesto uctievania".

Reply:
> Chrám zahŕňa aj kostol. Avšak `place_of_worship` môže byť aj na kaplnke, kríži, alebo hociakom mieste kde sa modlí. Takže ideálne len prípadne preformulovať "miesto uctievania".

chrám/svätyňa? To prve pre vacsie sakralne stavby, to druhe rozne miesta popri ceste apod.

Reply:
asi ano. v cs osm wiki to maju "Všechny sakrální objekty, tedy místa náboženských obřadů..."